src/models/schemaValidation.ts

```diff
@@ -0,0 +1,45 @@
+const Joi = require('@hapi/joi');
```

Comment:
I think this package is deprecated https://www.npmjs.com/package/@hapi/joi Use joi instead of @hapi/joi